packages/web/src/routes/session.tsx

```diff
@@ -97,20 +98,31 @@ function SessionPage() {
     <CellsProvider cells={session.cells}>
       <PackageJsonProvider session={session} channel={channel}>
         <TsConfigProvider session={session} channel={channel}>
-          <Session session={session} channel={channel} srcbooks={srcbooks} config={config} />
+          {/* <Session */}
+          {/*   session={session} */}
+          {/*   channel={channel} */}
+          {/*   srcbooks={srcbooks} */}
+          {/*   config={config} */}
+          {/* /> */}
+          <Session readOnly session={session} srcbooks={srcbooks} config={config} />
```

Comment:
Perhaps we could make this an env var in the short term?

Reply:
I added `VITE_DEBUG_RENDER_SESSION_AS_READ_ONLY` - I don't know if there's ever a use case where this would actually be used outside of development (maybe end to end testing?) but having a way to test this in app before it makes its way to other external codebases is probably a good idea.

Reply:
Yeah I think this is just short term so we don't have to edit git-tracked files to test it / dev against it.